src/components/Menu/index.vue

```diff
@@ -33,19 +33,20 @@ export default {
     return {
       greeting: 'Some Menu' as string,
       menuProducts: products,
-      filterButtons: '' as string,
       selectedCategory: 'all',
     };
   },
   methods: {
-    removeDuplicateCategory(): void {
+    categories(): void {
```

Comment:
Duplicate name, we'll need a method to select category on button click, maybe rename it to selectCategory?